pkg/asset/agent/common/utility.go

```diff
@@ -0,0 +1,10 @@
+package common
+
+import (
+	"github.com/google/uuid"
+)
+
+// GenerateInfraEnvID returns a random infra env ID.
+func GenerateInfraEnvID() string {
```

Comment:
Is the plan to substitute this generated infraenv id with the one in pkg/asset/agent/image/ignition?

Reply:
Good catch! Yes, indeed I planned to use it in first place. Somehow, missed it. Fixed